### PR TITLE
Update bitwarden extension

### PR DESCRIPTION
**Pull request by undefinedzack**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/bitwarden/src/utils/identity.ts [hunk 1]
```diff
@@ -15,6 +15,7 @@ ${Object.entries(identity)
 
     return `| ${shortcutKey ? `${shortcutKey}.` : "&nbsp;"}\t**${label}** | ${value} |`;
   })
+  .filter((ele) => ele !== null)
```
Comment (jomifepe) [inline extensions/bitwarden/src/utils/identity.ts:18]: That's definitely missing 😅 

You can filter by any falsy actually.

Could you also make this change in the `utils/cards.ts` file? Kinda the same login and also has this bug.

```suggestion
  .filter(Boolean)
```

**Review by jomifepe — APPROVED** [review]
Nice catch, thanks for contributing 🙂

File: extensions/bitwarden/src/utils/identity.ts (hunk 1 above)
Comment (undefinedzack) [reply extensions/bitwarden/src/utils/identity.ts:18]: Sure will do that right away!

**Comment by pernielsentikaer** [comment]
Could you add a changelog entry @undefinedzack

**Comment by undefinedzack** [comment]
Added change-log @pernielsentikaer

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**